docs/tutorial.rst

```diff
@@ -0,0 +1,12 @@
+=========
+Tutorial
+=========
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Tutorial:
```

Comment:
this caption ends up with the word Tutorial twice for no reason.  Either use the caption to say something new or remove it?